cpp/src/cider/function/CMakeLists.txt

```diff
@@ -19,16 +19,16 @@
 
 set(function_source_files
     ../../cider/exec/template/DynamicWatchdog.cpp
```

Comment:
do we need this?

Reply:
SubstraitToEU still has some history components that not be removed.